src/main/java/com/github/krzysiekjodlowski/ox/BoardPrinter.java

```diff
@@ -0,0 +1,31 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Generates String representation of the game board.
+ * @author Krzysztof Jodlowski
+ */
+class BoardPrinter {
+    /**
+     * @param board Board object
+     * @return string representation of the Board
+     */
+    String getBoardRepresentation(final Board board) {
+        int boardSize = board.getSize();
+        int boardCapacity = board.getBoardCapacity();
+        StringBuilder stringBuilder = new StringBuilder();
+        FieldNumber currentFieldNumber;
+
+        for (int i = 1; i <= boardCapacity; i++) {
+            currentFieldNumber = new FieldNumber(i);
+            stringBuilder.append(
+                    board.containMove(currentFieldNumber)
+                            ? String.format("%4s",
+                            board.getMove(currentFieldNumber))
+                            : String.format("%4d", i));
+            if (i % boardSize == 0) {
+                stringBuilder.append("\n");
+            }
+        }
```

Comment:
Iteration on every element in 10k x 10k board can be slow. I have no idea how to speed it up ;/

Reply:
Probably usage of TreeMap, iterating over its keyset and generating unmarked fields would be faster (no "contain" call for each number). I'll consider it:)